pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.ddp.model.activity.instance.validation;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.instance.answer.DecimalAnswer;
+import org.broadinstitute.ddp.model.activity.instance.question.Question;
+import org.broadinstitute.ddp.model.activity.types.RuleType;
+
+import java.math.BigDecimal;
+
+/**
+ * A validation rule that checks numeric integer value is within an optional min/max range, inclusive.
+ */
+public class DecimalRangeRule extends Rule<DecimalAnswer> {
+
+    @SerializedName("min")
+    private BigDecimal min;
+
+    @SerializedName("max")
+    private BigDecimal max;
```

Comment:
What does the JSON serialization of an object of this class look like?

Reply:
Good point. This should be changed to the DecimalDef as well. Like for Answer